src/components/engagement/engagement.rules.ts

```diff
@@ -383,7 +383,9 @@ export class EngagementRules {
     );
     if (!validNextStatus) {
       throw new UnauthorizedException(
-        'This status is not in an authorized sequence',
+        `One or more engagements cannot be changed to ${startCase(
```

Comment:
Is this called from anywhere else?  Meaning, will changing this message inadvertently show it somewhere else where it doesn't make sense?

Reply:
It's "complicated" a bit. But very straight forward. Anytime there's a mutation to update the engagement or and handler call to update the engagement.

It's called only from the engagement service:
![image](https://github.com/SeedCompany/cord-api-v3/assets/4716734/f7a2f77c-25a6-43c9-b8c6-40c8eb83b98c)

Which is called by the handler and resolver:
![image](https://github.com/SeedCompany/cord-api-v3/assets/4716734/2ffa82af-b9d4-460f-aba0-a80f8128d87a)
<img width="997" alt="image" src="https://github.com/SeedCompany/cord-api-v3/assets/4716734/856d15df-8d60-4f2f-87d1-e58f1b8095a7">